packages/mux-audio/src/index.ts

```diff
@@ -23,6 +23,7 @@ type AttributeNames = {
   METADATA_VIDEO_ID: 'metadata-video-id';
   METADATA_VIDEO_TITLE: 'metadata-video-title';
   METADATA_VIEWER_USER_ID: 'metadata-viewer-user-id';
+  BEACON_COLLECTION_DOMAIN: 'beacon-collection-domain',
   BEACON_DOMAIN: 'beacon-domain';
```

Comment:
I don't think we need to keep things backwards-compatible here tbh. This is an unannounced, undocumented feature so I think it's just cleaner to drop all references to `beacon-domain`

Reply:
yeah I thought about that, too. Just figured we could sweep that up in a followup PR.